TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -0,0 +1,21 @@
+@objc public class TradeItSDK: NSObject {
+    private static var apiKey: String?
+    private static var environment: TradeitEmsEnvironments?
+
+    public static let launcher = TradeItLauncher()
+    private (set) public static var linkedBrokerManager: TradeItLinkedBrokerManager!
+    private (set) public static var marketDataService: TradeItMarketService!
+    private (set) public static var brokerCenterService: TradeItBrokerCenterService!
+
+    public static func configure(apiKey: String, environment: TradeitEmsEnvironments = TradeItEmsProductionEnv) {
+        if self.apiKey == nil {
```

Comment:
Sure. It's a little more obvious but also adds a variable - I think it's ok either way.